features/fixtures/ast.yaml

```diff
@@ -1,87 +1,112 @@
-_version: 2.1
-metadata:
-- name: "Format"
-  value: "1A"
+
+_version: "2.1"
```

Comment:
This changes the version from being a interpreted as a number to a string in the YAML parser. This could be a breaking change for anything comparing the version in an AST parsed from YAML.

Here's an example in Ruby:

```ruby
YAML.load_file('original.yaml')['_version'] > 2.0
=> true

YAML.load_file('later.yaml')['_version'] > 2.0
ArgumentError: comparison of String with 2.0 failed from (pry):13:in `>'
```

```bash
$ cat original.yaml
_version: 2.1
$ cat later.yaml
_version: "2.1"
```